plopfile.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Defines a plop (https://plopjs.com/) generator to scaffold out new components
+ *
+ * Run with `yarn run plop` or `npm run plop`
+ */
+export default function (plop) {
```

Comment:
```suggestion
 * Run with `yarn run plop` or `npm run plop`
 *
 * @param {import('plop').NodePlopAPI} plop
 */
export default function (plop) {
```

We don't include this file when type checking currently, but adding this hint still enables better autocomplete.